src/Controller/WebPageArchiveController.php

```diff
@@ -23,16 +25,18 @@ class WebPageArchiveController extends ControllerBase {
   /**
    * Constructs a new WebPageArchiveController object.
    */
-  public function __construct(QueueFactory $queue) {
+  public function __construct(QueueFactory $queue, QueueWorkerManagerInterface $queue_manager) {
     $this->queue = $queue;
+    $this->queueManager = $queue_manager;
```

Comment:
I added the declaration.  I also added a comment asking if this is necessary to begin with. We're not presently using these anywhere in the controller. The only references to the queue and queue manager are actually inside static methods. But I'm opting to leave them in until we figure out what to do with the `viewRuns()` method, because we may want to interact with the queue on that page.  I'm just not 100% sure what that's gonna look like yet.